package.json

```diff
@@ -25,17 +25,23 @@
   "devDependencies": {
     "@types/enzyme": "^3.1.15",
     "@types/enzyme-adapter-react-16": "^1.0.3",
+    "@typescript-eslint/eslint-plugin": "^1.1.1",
+    "@typescript-eslint/parser": "^1.1.1",
     "codecov": "^3.1.0",
     "enzyme": "^3.8.0",
     "enzyme-adapter-react-16": "^1.7.1",
-    "jest-enzyme": "7.0.1",
+    "eslint-config-amo": "^1.10.0",
+    "eslint-plugin-amo": "^1.9.1",
+    "jest-enzyme": "^7.0.1",
     "prettier": "1.16.1",
     "pretty-quick": "^1.8.1",
     "type-coverage": "^1.6.2"
   },
   "scripts": {
     "build": "react-scripts build",
     "eject": "react-scripts eject",
+    "eslint": "eslint --ext .js --ext tsx src/",
```

Comment:
Does this still lint `.ts` files? It doesn't look like it.

Reply:
I thought we banned `.ts` files?

Reply:
Ah, yes we did! Amen. I filed https://github.com/mozilla/addons-code-manager/issues/75 so we don't forget to disallow `.ts` files.